numba/cuda/mathimpl.py

```diff
@@ -0,0 +1,246 @@
+import math
+import operator
+from llvmlite import ir
+from numba.core import types, typing
+from numba.core.imputils import Registry
+from numba.types import float32, float64, int64, uint64
+from numba.cuda import libdevice
+
+registry = Registry()
+lower = registry.lower
+
+
+booleans = []
+booleans += [('isnand', 'isnanf', math.isnan)]
+booleans += [('isinfd', 'isinff', math.isinf)]
+booleans += [('isfinited', 'finitef', math.isfinite)]
```

Comment:
I've checked this is correct, `isfinited` for double and `finitef` for float, imagine it's a historical artifact.

Reply:
Yes, it is odd!